cmd/sops/main.go

```diff
@@ -443,6 +444,55 @@ func main() {
 				return nil
 			},
 		},
+		{
+			Name:      "filestatus",
+			Usage:     "check the status of the file, returning encryption status",
+			ArgsUsage: `file`,
+			Flags:     []cli.Flag{},
+			Action: func(c *cli.Context) error {
+				// _, err := config.FindConfigFile(".")
+				// if err != nil {
+				// 	return common.NewExitError(err, codes.ErrorGeneric)
+				// }
```

Comment:
Should this command check for the config file?

Reply:
Not for now, at least, since we aren't using anything from the config file

Reply:
I'll remove this then.